### PR TITLE
Add direct installation links

### LINKED ISSUE #595: Put direct links to installation files in docs
All three recent user test users were slightly confused by https://nwb-guide.readthedocs.io/en/latest/installation.html

1. `setup.exe` does not exist
2. Which mac version do I pick?
3. Where are the files?

Now that we are closer to a full release, I suggest we put direct links to the installation files in the docs. We will need to manually update this for each release to point to new files.

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
fix #595 

Though, @garrettmflynn, is there a way you can remove the vX.Y.Z part from the file names? That's the only thing preventing us from using the `latest` tag and never having to update this to get it to work for latest releases (which I  just know is going to slip up one day)

### CODE REVIEW

**Comment by garrettmflynn** [comment]
Yep! We can add this to the `build` property of the `package.json` file: `artifactName": "NWB-GUIDE-${arch}.${ext}"`. Apparently `${arch}` and the leading dash will be removed if not present (e.g. for non-M1 Mac builds).

**Comment by CodyCBakerPhD** [comment]
> Yep! We can add this to the build property of the package.json file: artifactName": "NWB-GUIDE-${arch}.${ext}". Apparently ${arch} and the leading dash will be removed if not present (e.g. for non-M1 Mac builds).

Coolio; we'll confirm that once next release is cut and update docs to match at that point and it should be taken care of automatically thereafter

File: docs/installation.rst [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+
+Download the `NWB-GUIDE-vX.Y.Z.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.exe>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
 
-Run the `setup.exe` file and follow all instructions.
+MacOS - M1 Silicon
+------------------
 
-MacOS
----------------------------
+Download the `NWB-GUIDE-vX.Y.Z-arm64.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.exe>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
```
Comment (bendichter) [inline docs/installation.rst:18]: I think these links are incorrect. .exe files aren't generally used by macs.

File: docs/installation.rst [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
 Windows
-----------------------
+-------
+
+Download and run the `NWB-GUIDE-Setup-vX.Y.Z.exe <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.exe>`_ file and follow all instruction prompts.
+
+MacOS - Intel
+-------------
+
+Download the `NWB-GUIDE-vX.Y.Z.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.exe>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
```
Comment (rly) [inline docs/installation.rst:13]:
```suggestion
Download the `NWB-GUIDE-vX.Y.Z.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.dmg>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
```

File: docs/installation.rst (hunk 1 above)
Comment (rly) [inline docs/installation.rst:18]:
```suggestion
Download the `NWB-GUIDE-vX.Y.Z-arm64.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-0.0.13-arm64.dmg>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
```

File: docs/installation.rst [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
+
+Download and run the `NWB-GUIDE-Setup-vX.Y.Z.exe <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-Setup-0.0.13.exe>`_ file and follow all instruction prompts.
+
+MacOS - Intel
+-------------
+
+Download the `NWB-GUIDE-vX.Y.Z.dmg <https://github.com/NeurodataWithoutBorders/nwb-guide/releases/latest/download/NWB-GUIDE-0.0.13.dmg>`_ file, which should prompt you to move it into your 'Applications' folder in order to run.
 
-Run the `setup.exe` file and follow all instructions.
+MacOS - M1 Silicon
```
Comment (rly) [inline docs/installation.rst:15]:
```suggestion
MacOS - Apple Silicon
```
I assume the arm64 build will work on M2 and M3 as well.

Comment (garrettmflynn) [reply docs/installation.rst:15]: The last release initializes well on my M2

**Review by rly — APPROVED** [review]

**Review by rly — APPROVED** [review]

**Pull request merged**